tutorials/micro/micro_tflite.py

```diff
@@ -0,0 +1,219 @@
+# Using the buffer, transform into a tflite model python object
+try:
+    import tflite
+    tflite_model = tflite.Model.GetRootAsModel(tflite_model_buf, 0)
+except AttributeError:
+    import tflite.Model
+    tflite_model = tflite.Model.Model.GetRootAsModel(tflite_model_buf, 0)
+
+######################################################################
+# Print out the version of the model
+version = tflite_model.Version()
+print ("Model Version: " + str(version))
+
+
+######################################################################
+# Setup the device config which is what will be used to communicate
+# with the microcontroller (a STM32F746 Discovery board)
+TARGET = 'c -device=micro_dev'
+dev_config = micro.device.arm.stm32f746xx.generate_config("127.0.0.1", 6666)
+
+
+######################################################################
+# Parse the python model object to convert it into a relay module
+# and weights
+# It is important to note that the input tensor name must match what
+# is contained in the model.
+# If you are unsure what that might be, this can be discovered by using
+# the visualize.py script within the Tensorflow project.
+# See : How do I inspect a .tflite file? https://www.tensorflow.org/lite/guide/faq 
+input_tensor = "dense_4_input"
+input_shape = (1,)
+input_dtype = "float32"
+
+mod, params = relay.frontend.from_tflite(tflite_model,
+                                         shape_dict={input_tensor: input_shape},
+                                         dtype_dict={input_tensor: input_dtype})
+
+######################################################################
+# You'll need to uncomment the following blocks of code for the 
+# example to run on device.
+# Next with the dev_config, we establish a micro session and create
+# a context
+#
+# .. code-block:: python
+#
+# with micro.Session(dev_config) as sess:
+#   ctx = tvm.micro_dev(0)
+
+######################################################################
+# Now we create a build config for relay. turning off two options
+# and then calling relay.build which will result in a C source
+#
+# .. code-block:: python
+#
+#    disable_vectorize = tvm.target.build_config(disable_vectorize=True)
+#    disable_fusion = relay.build_config(disabled_pass={'FuseOps'})
+#    with disable_vectorize, disable_fusion:
+#        graph, c_mod, params = relay.build(mod, target=TARGET, params=params)
+
+######################################################################
+# With the c_mod that is the handle to our c sourcecode, we create a
+# micro module, followed by a compiled object which behind the scenes
+# is linked to the microTVM runtime for running on the target board
+# .. code-block:: python
+#
+#    micro_mod = micro.create_micro_mod(c_mod, dev_config)
+#    mod = graph_runtime.create(graph, micro_mod, ctx)
+
+######################################################################
+# Pass the weights to get ready to do some inference
+# .. code-block:: python
```

Comment:
Please put an empty line above the RST directive, otherwise RST can't recognize this directive.

Again, try running `make html` locally.